### DISCUSSION TITLE
Nowplaying showing radio station info

### DISCUSSION THREAD

**Discussion by jamesc2151** [ampache/ampache#3560] [Ideas]
is there a what that now playing can grab radio info for mpd play back from the state file or use a custom css file for the now playing area ???

....   Mine looks like this and the last few lines would be grate to display..

Start of file
____________________________________________________________________
sw_volume: 100
audio_device_state:1:EchoDot
state: play
current: 0
time: 116.604000
random: 1
repeat: 0
single: 0
consume: 0
crossfade: 0
mixrampdb: 0.000000
mixrampdelay: -1.000000
playlist_begin
song_begin: http://15313.live.streamtheworld.com/WGLFFM_SC?tdtok=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiIsImtpZCI6ImZTeXA4In0.eyJpc3MiOiJ0aXNydiIsInN1YiI6IjIxMDY0IiwiaWF0IjoxNjY5ODA3MDIyLCJ0ZC1yZWciOmZhbHNlfQ.FkLYH_MX_iTtpF2fWVw13bMjlm2tpn9qJ53cIywF0fg&DIST=TuneIn&TGT=TuneIn&maxServers=2&us_privacy=1YNY&partnertok=eyJhbGciOiJIUzI1NiIsImtpZCI6InR1bmVpbiIsInR5cCI6IkpXVCJ9.eyJ0cnVzdGVkX3BhcnRuZXIiOnRydWUsImxhdCI6IjMwLjQ5MDgiLCJsb25nIjoiLTg0LjMxNTgiLCJpYXQiOjE2Njk4MDcwMjIsImlzcyI6InRpc3J2In0.CqReB680-QbM7RZLSIQm8JFrFD2F30cOoslG0uVnB18
**Title: The Clash - Should I Stay Or Should I Go
Name: Gulf 104**
song_end
playlist_end
__________________________________________________________________________________________
End of File 

Local play instance shows this info

![image](https://github.com/ampache/ampache/assets/25990001/aed6abf6-2965-466a-8995-9a3245944359)

**Comment by lachlan-00** [comment]
it will be an issue parsing the data. i'm assuming ampache isn't able to id it correctly and is returning the whole string for track_title

it was able to identify "Connor Smith" though so it's partially there

![image](https://github.com/ampache/ampache/assets/1305249/d0becd5c-6509-4673-9331-0f365dd6c019)

**Comment by jamesc2151** [comment]
So how about a way for it to grab the info from the radio broadcast selected to play and display that it's playing a Broadcast.... I'd do a screen shot for ya but I bought a new case for my Raspberry and I wound up shorting out the 3.3 volt side or it. ( Replacement here tomorrow tho.. Can't Live without it so I'll post one then for ya) Also got a question about entrys on that screen as Edits are not being saved?? Will check that again too as I had just loaded the latest ver. 
Thanks

![image](https://github.com/ampache/ampache/assets/25990001/7e97601f-2a86-46bf-b207-314e0df46352)

Radio Station edit screen
![image](https://github.com/ampache/ampache/assets/25990001/a4480f4a-e853-48fe-a6b2-8f69d87a5e68)
